### ISSUE TITLE
question about 'SD Features'

### ISSUE THREAD

**Issue by develop-productivity** [diffusion-classifier/diffusion-classifier#9]
I'm looking forward to seeing you open up the implementation code for the 'SD Features' model mentioned in the paper. Because I think SD features is a research area.

**Comment by Darkbblue** [comment]
Hey ive implemented the code at https://github.com/Darkbblue/diffusion-feature. Though it can run on my server, i havent tested the extracted features on image classification tasks. Maybe you want to give it a try?

**Comment by develop-productivity** [comment]
Thanks. I'll take a look

**Comment by Yeez-lee** [comment]
> Thanks. I'll take a look

Hi, did you do the tests?

**Comment by develop-productivity** [comment]
> > Thanks. I'll take a look
> 
> Hi, did you do the tests?

No, I didn't do the tests, but I made some modification to fit my project.
